includes/class-wcmp-postcode-fields.php

```diff
@@ -738,7 +738,7 @@ public function validate_address_fields($address, $errors)
         }
 
         if (self::isCountryWithSplitAddressFields($address['shipping_country'])
-            && array_key_exists('ship_to_different_address', $address)
+            && ! empty( $address['ship_to_different_address'] )
```

Comment:
Minor code style detail:
```suggestion
            && ! empty($address['ship_to_different_address'])
```